plugins/woocommerce/src/Internal/Admin/Loader.php

```diff
@@ -408,9 +407,16 @@ public static function add_component_settings( $settings ) {
 		// WooCommerce Branding is an example of this - so pass through the translation of
 		// 'WooCommerce' to wcSettings.
 		$settings['woocommerceTranslation'] = __( 'WooCommerce', 'woocommerce' );
-		// We may have synced orders with a now-unregistered status.
-		// E.g An extension that added statuses is now inactive or removed.
-		$settings['unregisteredOrderStatuses'] = self::get_unregistered_order_statuses();
+
+		if ( PageController::is_admin_page() && Features::is_enabled( 'analytics' ) ) {
```

Comment:
Finding out in p1740650640023039-slack-C0410KV3YLW where the settings exposed in this class are getting used (for testing and testing instructions).

Reply:
Concluded in favour of deprecations in the loader class